tools/machine-learning/mujoco/test_script.py

```diff
@@ -0,0 +1,47 @@
+import gymnasium as gym
```

Comment:
Is this file still needed? What does this do that `standup.py` doesn't do?

Reply:
Yea okay, removed